### DISCUSSION TITLE
How can I obtain the hkl of a plane when the Euler direction of this cystal and the direction of the plane in sample coordinate system is known?

### DISCUSSION THREAD

**Discussion by Dxc0000** [mtex-toolbox/mtex#1806] [Ask Anything]
as questioned

for example,
the crystal Euler direction is:
phi1       Phi         phi2
236.022 121.111 319.662
the direction of the plane is:
[0,1,9] in sample coordinate system

in this case, how can I obtain the hkl of this plane in the crystal?

**Comment by lfgmorales** [comment]
Hi

<https://mtex-toolbox.github.io/DefinitionAsCoordinateTransform.html>
Crystal Orientation as Coordinate Transformation | MTEX<https://mtex-toolbox.github.io/DefinitionAsCoordinateTransform.html>
mtex-toolbox.github.io<https://mtex-toolbox.github.io/DefinitionAsCoordinateTransform.html>
        [favicon.png] <https://mtex-toolbox.github.io/DefinitionAsCoordinateTransform.html>



Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 29 Apr 2023, at 11:34, Dxc0000 ***@***.***> wrote:

﻿

as questioned

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1806>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN2LEEFQCME2VKLJVB3XDTOANANCNFSM6AAAAAAXQDNJLU>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1806", "url": "https://github.com/mtex-toolbox/mtex/discussions/1806", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
